packages/stream/types.d.ts

```diff
@@ -40,8 +47,8 @@ export declare function makeStream<
   TReadReturn = undefined,
   TWriteReturn = undefined,
 >(
-  acks: AsyncQueue<IteratorResult<TRead, TReadReturn>>,
-  data: AsyncQueue<IteratorResult<TWrite, TWriteReturn>>,
+  acks: AsyncSpring<IteratorResult<TRead, TReadReturn>>,
+  data: AsyncSink<IteratorResult<TWrite, TWriteReturn>>,
```

Comment:
I imagine @mhofman will be interested in this relaxation of the signature of `makeStream`. This is a remnant of the pubsub work. Although we’re not pursuing that variety of pubsub, this does make the type signature of `makeStream` no more specific than necessary, which I think is good POLA.